apps/sms/js/recipients.js

```diff
@@ -109,6 +119,14 @@
     'type', 'carrier', 'isQuestionable', 'isInvalid', 'isLookupable'
   ];
 
+  Recipient.cast = function(wannabeRecipient) {
```

Comment:
Usually ```like``` postfix is used for such things(e.g. ```recipientLike```), but ```wannabe``` is funnier, so I'm ok with it :)